### PR TITLE
Nfs3

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

**Comment by rminnich** [comment]
sorry about this, but the nfs3 performance is just a ton better than 9p, across the board. I have found no case in which 9p is remotely competitive :-(

**Comment by paulmenzel** [comment]
> nfs3 is available than 9p;

more available?

File: client/osnfs.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+type COS struct {
+	billy.Filesystem
+}
+
+// Chmod changes mode
+func (fs COS) Chmod(name string, mode os.FileMode) error {
+	return os.Chmod(fs.Join(fs.Root(), name), mode)
+}
+
+// Lchown changes ownership
```
Comment (paulmenzel) [inline client/osnfs.go]: Extend for ignorant like me: … but does not dereference symbolic links

**Comment by rminnich** [comment]
Paul! I'm glad you are here :-), thanks!

**Comment by Lencerf** [comment]
I tried locally and I am seeing a lot of errors of `call to 0x7b29c0 failed: I/O error: path outside base dir`.

**Comment by rminnich** [comment]
It's waiting for this to go in: https://github.com/go-git/go-billy/pull/48

File: client/osnfs.go (hunk 1 above)
Comment (rminnich) [reply client/osnfs.go]: done

**Comment by rminnich** [comment]
> > nfs3 is available than 9p;
> 
> more available?

done

**Comment by rminnich** [comment]
try again, that merged

**Comment by Lencerf** [comment]
It seems to work, but I am still seeing a lot of `call to 0x7b40a0 failed: No such file or directory: file does not exist`.

**Comment by rminnich** [comment]
That was this one, also merged:
https://github.com/willscott/go-nfs/pull/133/files

I will do a go mod tidy

**Comment by rminnich** [comment]
do a go mod tidy

**Comment by Lencerf** [comment]
I did 

```sh
go get github.com/go-git/go-billy/v5@b7d9906
go get github.com/willscott/go-nfs@91bc389
```

and it worked.

**Review by Lencerf — APPROVED** [review]

**Comment by rminnich** [comment]
Sorry, 9p2000!

**Pull request merged**